src/components/ShowPage.js

```diff
@@ -20,16 +20,40 @@ import Error from './Error';
 import ShowRespond from './ShowRespond';
 import ShowResults from './ShowResults';
 import ShareModal from './ShareModal';
+import Modal from 'react-modal';
+import './ReactModalOverride.scss';
+
 
 import sharedStyles from './SharedStyles.module.scss';
 import styles from './ShowPage.module.scss';
 import clipboardIcon from '../clipboard-regular.svg'; // https://fontawesome.com/license
 
+// Modal.setAppElement('#root');
 class ShowPageComponent extends Component {
-  state = {
-    pendingSubmission: null, // Shape: { showToSave: Show!, name: String, email: String, responses: [Date]! }
-    hasSetName: false,
-  };
+  constructor(props) {
+    super();
+    this.state = {
+      pendingSubmission: null, // Shape: { showToSave: Show!, name: String, email: String, responses: [Date]! }
+      hasSetName: false,
+      modalIsOpen: props.isModalOpen || false,
```

Comment:
This prop is never set. I can see why it'll be used though. Was it added for futureproofing?

Reply:
This is so that if in the future you want to open the modal on render if coming to this page from lets say the create page then you can